2-library/unit-testing/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.6core.lib</groupId>
+    <artifactId>unit-testing</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+<!--    <version>1.0-SNAPSHOT</version>-->
+
+    <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
```

Comment:
Do we need to have both? 
`junit-jupiter` and `junit-jupiter-api`?

junit-jupiter aggregates itself junit-jupiter-api and other depencency.